standards.iso.org/iso/19115/resources/transforms/ISO19139/mapping/core.xsl

```diff
@@ -433,9 +433,9 @@
       </xsl:element>
     </xsl:element>
   </xsl:template>
-  <xsl:template match="gmi:objectiveOccurance" mode="from19139to19115-3">
+  <xsl:template match="gmi:objectiveOccurrence" mode="from19139to19115-3">
     <!-- This element is mis-spelled in the 19115-2 schema -->
-    <xsl:element name="mac:objectiveOccurence">
+    <xsl:element name="mac:objectiveOccurrence">
```

Comment:
In the UML for ISO19115-2 Edition 1 and Edition 2, and in ISO19115-3 Acquisition information - Imagery package , the association from MI_Objective to MI_Event is named 'objectiveOccurence', but in the XML implementation at http://standards.iso.org/iso/19115/-2/gmi/1.0/acquisitionInformation.xsd, it's spelled 'objectiveOccurance'.  The spelling 'gmi:objectiveOccurance' must be used in the xsl file, since that's what's in the xml schema. Correcting the typo to match what's in the UML requires an amendment to the spec.
The ISO19115-3 xml implementation (http://standards.iso.org/iso/19115/-3/mac/1.0/acquisitionInformationImagery.xsd)  has the typo from the UML (mac:objectiveOccurence)
The xsl has the correct (mis)spellings, don't change

Reply:
Hello all, sorry for delay getting back on this... just too busy these days.

In general the discussion is correct - we can change annotations but not mis-spelled elements (we could do a corrigendum, but may be too late)...

Also wanted to make sure that everyone understands that the annotated schema is not the official schema and is not supported in any way by ISO TC211. It was a project a few people started with the hope that it would be useful and that the community would help support it. Thanks so much for help with the spelling...

Another note - the new and complete schemas have now been uploaded to standards.iso.org. There are a number of new namespaces that have Version 2 directories. I will update those in the annotated schemas soon. We may need a mechanism for transferring annotations across to the new namespace versions... Will also think about that...